draft-ietf-oauth-token-binding.xml

```diff
@@ -1165,6 +1145,28 @@
           <spanx style="verb">client_access_token_token_binding_supported</spanx>
           metadata parameter).
         </t>
+        <t>
+          Some web-based confidential clients implemented as distributed nodes may
+          be perfectly capable of implementing access token binding (if the access
+          token remains on the node it was bound to, the token binding keys would
+          be locally available for that node to prove possession), but may struggle with refresh token
+          binding due to an inability to share token binding key material between nodes.
+          As confidential clients already have credentials which are required to
+          use the refresh token, and those credentials should only ever sent over
+          TLS server-to-server between the client and the Token Endpoint, there is still value in
+          token binding access tokens without token binding refresh tokens.
+          Authorization servers SHOULD consider supporting access token binding
+          without refresh token binding for confidential web clients as there are
+          still incremental security benefits to do so.
```

Comment:
I feel like the word 'incremental' here, while true, somewhat downplays the full benefits. Not sure of a better way to say it though...